modules/kubernetes/grafana-agent/charts/grafana-agent-extras/values.yaml

```diff
@@ -1,24 +1,16 @@
 image: grafana/agent:v0.21.2
 replicas: 2
-environment: dev
+
+environment: ""
 clusterName: aks1
-credentialsSecretName: grafana-agent-credentials
+cloudProvider: ""
 
 remote:
-  metricsUrl: ""
   logsUrl: ""
-  tracesUrl: ""
-
-# The resource values specified for agent will apply both to grafana-agent and grafana-agent-logs
-# since it is not possible to configure different values for those two
-agent:
-  resources:
-    requests:
-      cpu: 25m
-      memory: 100Mi
```

Comment:
Why no resource definition?

Reply:
Oops, removed by accident. I'm thinking that maybe we should change to not hard code the values?